modules/monitoring-common-terms.adoc

```diff
@@ -54,6 +54,9 @@ Kubernetes scheduler allocates pods to nodes.
 labels::
 Labels are key-value pairs that you can use to organize and select subsets of objects such as a pod.
 
+Metrics Server::
+The Metrics Server monitoring component collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs, which frees the core platform Prometheus stack from handling this functionality.
```

Comment:
I saw (MS) after metrics server in another module in this PR, should that be included here too?

Reply:
Good catch! The `MS` in the other location is to help identify what "MS" stands for in the architecture diagram that's included above that mention. So I think it's fine to leave it out here.